api/client/swarm/opts.go

```diff
@@ -133,17 +130,32 @@ func (o *AutoAcceptOption) Policies(secret *string) []swarm.Policy {
 	policies := []swarm.Policy{}
 	for _, p := range defaultPolicies {
 		if len(o.values) != 0 {
-			p.Autoaccept = o.values[string(p.Role)]
+			if _, ok := o.values[string(p.Role)]; ok {
+				p.Autoaccept = true
+			} else {
+				p.Autoaccept = false
+			}
 		}
 		p.Secret = secret
 		policies = append(policies, p)
 	}
 	return policies
 }
 
+// IsPresent returns whether the key exists in the set or not
+func (o *AutoAcceptOption) IsPresent(key string) bool {
+	_, c := o.values[key]
+	return c
+}
+
+// AddValue adds a value to the set of auto-accepted nodes
+func (o *AutoAcceptOption) AddValue(value string) {
```

Comment:
Please unexport `IsPresent` and `AddValue` since they aren't meant to be used outside the package.

You might also consider removing `AddValue` and doing the map key assignment directly.